http/robots/robots.go

```diff
@@ -0,0 +1,19 @@
+// Package robots provides a way of dealing with robots exclusion protocol
+package robots
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/apex/up"
+)
+
+// New robots middleware.
+func New(c *up.Config, next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if os.Getenv("UP_STAGE") != "production" {
```

Comment:
Works for me! Only thing I'd tweak for now is:

```
if os.Getenv("UP_STAGE") == "production" {
  return return next
}

return otherMiddlewareHere
```

Or we could delegate that decision to the handler.go so it's just never added for prod

Reply:
Yeah, that's much better. Incapsulated it in the middleware for now